capstone/src/main/resources/static/user.js

```diff
@@ -172,38 +138,34 @@ function hideFields(selectField, universityField) {
 
 /* get the saved events for individual users */
 function getIndividualEvents() {
-  var userType = sessionStorage.getItem("user-type");
-  if (userType == null) {
-    getUserType(true);
-  }
-  if(userType == "individual") {
-    fetch('get-' + userType).then(response => response.json()).then((data) => {
-      const eventDiv = document.getElementById("saved-events");
-      data[0].savedEvents.forEach((event) => eventDiv.appendChild(createSavedEventElement(event, false, false)));
-      displayMain(true);
-    });
-  } else {
-    displayMain(false);
-  }
-
+  fetch('user-info').then(response => response.json()).then((data) => {
+    var userType = data.userType;
+    if(userType == "individual") {
```

Comment:
It seems wasteful to do a call to `user-info` endpoint just to check that the current user is an individual. Since "individual" is in the name of the function, you can put the burden on the caller to make sure that they are only calling it for individuals.